src/main/java/nl/novi/TechItEasy/repositories/TelevisionRepository.java

```diff
@@ -0,0 +1,9 @@
+package nl.novi.TechItEasy.repositories;
+
+import nl.novi.TechItEasy.models.Television;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
Deze annotatie is hier niet nodig. We extenden hier van de JpaRepository. Daar wordt al aangegeven dat het om een repository gaat.